symposion/reviews/views.py

```diff
@@ -112,9 +138,9 @@ def review_section(request, section_slug, assigned=False):
         assignments = ReviewAssignment.objects.filter(user=request.user).values_list("proposal__id")
         queryset = queryset.filter(id__in=assignments)
 
-    queryset = queryset.select_related("result").select_subclasses()
+    queryset = queryset.select_related("result", "kind", "speaker__user", "tags").select_subclasses()
 
-    proposals = proposals_generator(request, queryset)
+    proposals = list(proposals_generator(request, queryset))
```

Comment:
It actually did end up returning a list. I'm going to update the name and comment and remove the redundant `list()` calls on its return value.